Source/Resources/en.lproj/Localizable.strings

```diff
@@ -15,7 +15,9 @@
 "ExpiryDate" = "ExpiryDate";
 "ExpiryDateFormat" = "Format is MM/YY";
 "ExpiryDatePlaceholder" = "MM/YY";
-"ExpiryDateErrorMessage" = "Missing Expiry Date";
+"ExpiryDateErrorMessageInvalid" = "Invalid Expiry Date";
+"ExpiryDateErrorMessageMissing" = "Please enter a valid Expiry Date";
```

Comment:
ExpiryDateErrorMessageMissing and ExpiryDateErrorMessageInvalid both mean same, Can we club them into one ? . Or is there a reason for it to be 2 different strings ?

Reply:
@patrick-hoban-cko please advice

Reply:
As per https://checkout.atlassian.net/browse/PIMOB-1274, _Please enter a valid Expiry Date_ appears when the user has not entirely filled out the month or year, for example - if they put 06/2

_Invalid Expiry Date_ should never appear in reality, as it's only a placeholder for error scenarios that should be handled by either the keypad, or smart month/year logic.